### PR TITLE
working on lsst versions of medsifier and metadetect

### PR DESCRIPTION
**Pull request by esheldon**

### CODE REVIEW

File: metadetect/lsst_medsifier.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        # run the deblender
+        deblend_task.run(exposure, sources)
+
+        self.sources = sources
+
+
+def get_exposure_from_obs(obs):
+    ny, nx = obs.image.shape
+
+    scale = obs.jacobian.scale
```
Comment (beckermr) [inline metadetect/lsst_medsifier.py]: I think we need to grab the full jacobian here.

Comment (esheldon) [reply metadetect/lsst_medsifier.py]: that code is unused, removed in latest push

File: examples/make-condor.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
+import os
```
Comment (beckermr) [inline examples/make-condor.py]: Thank you @esheldon !

Comment (esheldon) [reply examples/make-condor.py]: I added the script it runs too, but you'll still need to modify paths

**Comment by esheldon** [comment]
gonna merge this, it does what we need for now

**Pull request merged**